module/validation/receipt_validator_test.go

```diff
@@ -881,22 +829,9 @@ func (s *ReceiptValidationSuite) TestValidateReceiptResultHasEnoughReceipts() {
 	}
 	s.PersistedResults[result0.ID()] = result0
 
-	candidateReceipts := []*flow.ExecutionReceiptMeta{receiptB.Meta()}
```

Comment:
I think it is still worthwhile to test both valid cases:
- Proposer includes result and 1 receipt in block
- Proposer includes result and multiple receipts in block